### DISCUSSION TITLE
Issue in importing spacy | getting an error - No module named 'thinc.api'

### DISCUSSION THREAD

**Discussion by aman9801** [explosion/spaCy#10853] [Help: Coding & Implementations] [answered]
I am working in a Jupyter notebook
```
import spacy
```
Getting this error:
```
ModuleNotFoundError                       Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 import spacy

File ~/opt/anaconda3/lib/python3.9/site-packages/spacy/__init__.py:11, in <module>
      8 setup_default_warnings()  # noqa: E402
     10 # These are imported as part of the API
---> 11 from thinc.api import prefer_gpu, require_gpu, require_cpu  # noqa: F401
     12 from thinc.api import Config
     14 from . import pipeline  # noqa: F401

ModuleNotFoundError: No module named 'thinc.api'
```
### Environment:

```
pip list | grep spacy
spacy            3.3.0
spacy-legacy     3.0.9
spacy-loggers    1.0.2

pip list | grep thinc
thinc                  8.0.16
thinc-apple-ops        0.0.6

python --version
Python 3.9.12
```

**Platform:** macOS-12.3.1-arm64-arm-64bit

**Comment by polm** [comment, marked as answer]
Huh, that's weird. That version of thinc looks fine. 

How did you install spaCy? Did you restart your kernel after installing it? Are you sure the environment in your notebook is the same one you ran `pip list` in?

**Reply by aman9801** [reply]
Yes, I did restart the kernel
I installed it using this - 
```
pip install -U 'spacy[apple]'
```

Yes, environment is same.
<img width="1140" alt="Screenshot 2022-05-26 at 2 37 33 AM" src="https://user-images.githubusercontent.com/21023241/170432133-e28a149b-4e88-43bb-ab17-d5e6c34bacf0.png">

**Reply by polm** [reply]
That's weird, I'm not really sure what could cause that error.

Here are some things you can check or try:

1. Can you `import thinc` directly? 
2. Uninstall and reinstall `thinc`
3. Use a blank environment and install `spacy` from scratch

Also be sure to follow the install instructions [here](https://spacy.io/usage) - it looks like you're using conda but installing with pip. That's usually not a problem but it can be, especially on M1 Macs, where it's possible to end up with packages for different CPU architectures. 

Also, if you provide any other information, please copy and paste text as text, not as images.

**Comment by aman9801** [comment]
This is resolved. I installed it in a new environment and it's working fine.

**Reply by polm** [reply]
Glad you figured it out! Installing in a new environment is usually the best way to resolve install related issues that are hard to figure out.
